docs/CONNECTOR_PROTOCOL.md

```diff
@@ -1,5 +1,4 @@
 # Connector Protocol
-**Version 1, Jul 25 2022**
```

Comment:
why do we remove this? even though it's versioned I think it would be great to keep the version here and start to increment the version protocol when we do changes that are changing the behavior

Reply:
>even though it's versioned

This is exactly the reason I removed this. 

It's hard for me to predict the date. I have to update this again before I merge the PR